packages/core/src/new-api/types/provider.ts

```diff
@@ -15,6 +13,6 @@ export interface RequestArguments {
  *
  * @beta
  */
-export interface EIP1193Provider extends EventEmitter {
+export interface EIP1193Provider {
```

Comment:
We don't use that interface, so we can avoid it, making mocking this way simpler.